software-templates/migrate-repo-to-enterprise/template.yaml

```diff
@@ -0,0 +1,300 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: GitHub-Repo-Transfer
+  title: Transfer a repository between organizations
+  description: Transfer a repository from the bcgov GitHub organization to the bcgov-public GitHub enterprise organization
+spec:
+  owner: "developer-experience"
+  type: service
+
+  parameters:
+    - title: Transfer a repository
+      required:
+        - repoToTransfer
+        - owner
+        - destination
+      properties:
+        repoToTransfer:
+          title: Repository to transfer
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            requestUserCredentials: # this is super important - it causes the user to be prompted to log in to GitHub and/or authorize the required GH scopes.
+              secretsKey: USER_OAUTH_TOKEN
+              additionalScopes:
+                github:
+                  - workflow
+            allowedHosts:
+              - github.com
+            allowedOwners:
+              - bcgov
+        destination:
+          title: Destination GitHub Organization
+          type: string
+          description: GitHub organization where the repo will be transferred to.
+          enum: 
+            - bcgov-public
+          enumNames: 
+            - "bcgov-public"
+        owner:
+          title: GitHub team (required)
+          type: string
+          description: The team in the bcgov-public GitHub Organization that maintains the repo.
+          ui:field: OwnerPicker
+          ui:options:
+              allowArbitraryValues: false
+              catalogFilter:
+                - kind: [ Group ]
+                  metadata.namespace: bcgov-public
+
+    - title: Provide information about your product/service/component
+      required:
+        - title
+        - description
+        - type
```

Comment:
This may confuse users...what can we default this to so it won't break anything? or can it be optional? Might need additional instructional text.

Reply:
@sheaphillips  - If I update 'type' to "Product category" and put in the description you suggested below, does that help clarify it?

This is for the catalog-info.yaml file. In our previous templates it was defaulted to 'documentation'. I can default it to something here. My only concern is people might not know they can change it?

Reply:
let's see what comes out of usability testing as we work through this, but I'm anticipating we'll eventually need a 'dev hub' guide that has a section about catalog info files, and expected values etc